.github/workflows/add_to_project.yml

```diff
@@ -0,0 +1,20 @@
+name: Add bugs to bugs project
+
+on:
+  issues:
+    types:
+      - opened
+
+jobs:
+  add-to-project:
+    name: Add issue to project
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/add-to-project@v1.0.2
+        with:
+          # You can target a project in a different organization
+          # to the issue
+          project-url: https://github.com/orgs/nightscout/projects/2
+          github-token: ${{ secrets.ADD_TO_PROJECT_PAT }}
+          labeled: bug, needs-triage
```

Comment:
I looked at the documentation for actions/add-to-project, and the current configuration will only add issues opened with labels bug OR needs-triage. Meaning enhancements are not included (not sure if we use any other labels). 

https://github.com/actions/add-to-project?tab=readme-ov-file#examples

Reply:
correct, this only targets bug issues for now.
If we do the features here aswell we need a different strategy for that anyway.

Reply:
Ok, I guess this is fine for now. 

Also noticing use of separate GH token secrets here and in PR 323. I guess they could be the same, so the secrets doesn’t have to be entered in duplicate?

Reply:
Nope, different PAT's with different rights They are already in the secret cage